src/api/local-storage.js

```diff
@@ -0,0 +1,86 @@
+class localStorageApi {
+ 
+  static createUser(newUser) {
+      localStorage.setItem("newUser", JSON.stringify(newUser));
+      var existingUsers = localStorageApi.getUsers();
+      existingUsers.push(newUser);
+      localStorage.setItem("allStoredUsers", JSON.stringify(existingUsers));
```

Comment:
To make it easier on yourself later could you create an object of database locations? Something like:

```
const databaseEndpoints = {
  ALL_USERS: 'allStoredUsers'
}
```

And then you could reference it for something like:

```
 localStorage.setItem(databaseEndpoints.ALL_USERS, JSON.stringify(existingUsers));
```

Reply:
will localStorage accept the syntax?

Reply:
looks like it does